frontend/src/components/podcast/episode-table/episode-table.tsx

```diff
@@ -0,0 +1,102 @@
+import { useMemo, useTable } from 'hooks/hooks';
```

Comment:
we need to create a new common component for the table because we will reuse it 

`frontend/components/common/table`

it should have 2 props - `columns` and `data` 

```tsx
type Props = {
  columns: Column[];
  data: unknown[];
}
```